pkg/kubelet/status/status_manager.go

```diff
@@ -471,6 +471,19 @@ func (m *manager) syncBatch() {
 	}
 }
 
+func (m *manager) getTPClient(tenant string) clientset.Interface {
```

Comment:
this can be a util function in the utils folder to avoid dups in multiple manager components.